docs/_build/html/_sources/home.rst.txt

```diff
@@ -52,20 +52,78 @@ To make any changes in the code follow the below steps :
   :align: center
   :alt: Alternative text
 
-- To add some different image go to  `style.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/style.css>`__  and search :python:`.parallax1` and change the url. 
+- To add some different image - Go to  `style.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/style.css>`__  and search :python:`.parallax1` and change the url. 
 
 .. note::
    There are three Parallax in the home page.
 
 
-4. **Our Initiatives**:
+4. **Our Initiatives**:  This section describes about the different initiatives taken by girlscript.
```

Comment:
**taken by your GirlScript Chapter
Keep in mind this documentation is supposed to be a guide addressing various Chapter members in second person

Reply:
Ohhhhh

Reply:
okay sir